simple-wp-helpdesk/assets/swh-api-keys.css

```
@@ -0,0 +1,295 @@
+	border-color: var( --swh-color-success-accent, #28a745 ) !important;
+	color: #fff !important;
+}
+
+/* --- aria-live region (visually hidden) ------------------------------- */
+
+.swh-apikey-modal-live-region {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	overflow: hidden;
+	clip: rect( 0 0 0 0 );
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)